### ISSUE TITLE
Workbench: Enhance evaluation for NN regression

### ISSUE THREAD

**Issue by PetrToman** [encog/encog-java-core#92]
1. In Workbench, create .ega file for NN regression - using data: http://dione.zcu.cz/~toman40/encog/data9.zip
2. Execute task-full and "Stop all commands" when task-train begins.
3. "Normalize to training" `data9_eval.csv`.
4. Double click `data9_train.eg`, click "Train" -> select `data9_train.egb` for as "Training Set" and `data9_eval.egb` as "Validation Set" -> choose RPROP, ensure "Maximum Error Percent" = 3 and let the training run.
5. Close all tabs (saving the training if a popup appears), double click `data9.ega` and execute task-evaluate.
6. Check `Output:y` column in `data9_output.csv` -> it contains all zeroes.
7. Delete `data9_output.csv` and replace  `data9_eval.csv` by `data9_train.csv`.
8. Repeat step 5.
9. Check `Output:y` column in `data9_output.csv` -> it contains almost all zeroes. However, since the training error was ~2.6%, `Output:y` should contain values very close to those in `y` column (the training data are used now).

In contrast, if the steps above are taken using SVM regression, the `data9_output.csv` contains expected values - i.e. in the latter case, `Output:y` contains values close to `y`.
